packages/components/src/date-time/date.js

```diff
@@ -33,7 +38,15 @@ function DatePickerDay( { day, events } ) {
 			{ day.format( 'D' ) }
 			<ul className="components-datetime__date__day-bullets">
 				{ times( Math.min( events?.length, 3 ), ( i ) => (
-					<li key={ `bullet-${ i }` }> </li>
+					<li key={ `bullet-${ i }` }>
+						<VisuallyHidden>
+							{ _n(
+								'has one post',
+								'has some posts',
+								events.length
```

Comment:
I guess the problem here is that it's a generic UI component, but the message is specific, it's says "post". So maybe we should have a way to define messages like we do in FormTokenField.

Reply:
Could we make the message more generic? "Has published item(s)" or something?

Reply:
>  So maybe we should have a way to define messages like we do in FormTokenField.

I'm not sure how to implement a similar approach that `FormTokenField` here. It speaks depending on the action that the user does, is it correct? Add or delete a new token, search results, etc. To do it here, should we detect when the day is focused for instance, and then speaks the day events?

> Could we make the message more generic? "Has published item(s)" or something?

It'd simplify it considerably. Do you think it could work, Riad?

Reply:
> It'd simplify it considerably. Do you think it could work, Riad?

"published" doesn't sound generic enough. A generic thing I could think of is: There are x events